.editorconfig

```diff
@@ -6,11 +6,11 @@ root = true
 # Unix-style newlines
 # Remove end-of-file newlines
 # Soft indents (2 spaces)
-# Trim tailing whitespace
+# Don't trim trailing whitespace
 [*]
 charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 indent_style = space
 indent_size = 2
-trim_trailing_whitespace
+trim_trailing_whitespace = false
```

Comment:
Nvm, made a mistake (should be true)
On Thu, Sep 1, 2016 at 10:32 PM Kaitlin Hipkin <notifications@github.com>
wrote:

> In .editorconfig
> <https://github.com/HabitRPG/habitrpg/pull/7957#discussion_r77288964>:
>
> >  [*]
> >  charset = utf-8
> >  end_of_line = lf
> >  insert_final_newline = true
> >  indent_style = space
> >  indent_size = 2
> > -trim_trailing_whitespace
> > +trim_trailing_whitespace = false
>
> why false?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7957/files/47a4bd482497d7055e3a14307dc183b7e7c7fb33..0bce0b1a72a527af95b194b0ed752385166d5e75#r77288964>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_81-czfJd79X7V1CJrtIqeMfz6_O5ks5ql5i6gaJpZM4Jx4Ke>
> .
>